src/content/en/2020/seo.md

```diff
@@ -83,183 +84,227 @@ Response Code | All Devices | Mobile | Desktop
+To understand what search engines are able to effectively extract, we have broken out the components of that content and examined the incidence rate of those features between the mobile and desktop contexts. We also reviewed the disparity between mobile and desktop content. The mobile and desktop disparity is especially valuable because Google has moved to [_mobile-first indexing_](https://developers.google.com/search/blog/2020/03/announcing-mobile-first-indexing-for) (MFI) for all new sites and, as of March of 2021, will move to a _mobile-only index_ wherein content that does not appear within the mobile context will not be evaluated for ranking.
 
 
-#### Rendered vs Non-rendered Text Content
-The usage of Single Page Application (SPA) JavaScript technologies has exploded with the growth of the web. This design pattern introduces difficulties for search engine spiders because both the execution of JavaScript transformations at runtime and user interactions with the page after load can cause additional content to appear or be rendered.
+#### Rendered versus non-rendered text content
+{# TODO(editors): Update link to 2020 JS technologies section. #}
+The usage of Single Page Application (SPA) [JavaScript](../javascript) technologies has exploded with the growth of the web. This design pattern introduces difficulties for search engine spiders because both the execution of JavaScript transformations at runtime and user interactions with the page after load can cause additional content to appear or be rendered.
 
 Search engines encounter pages through its crawling activity, but may or may not choose to implement a second step of rendering a page. As a result, there may be disparities between the content that a user sees and the content that a search engine indexes and considers for rankings.
 
-We assessed word count in context of that disparity.
+We assessed word count as a heuristic of that disparity.
 
 {{ figure_markup(
   image="seo-visible-words-per-page-raw.png",
-  caption="Visible Words Per Page (raw)",
+  caption="Distribution of the number of raw words per page.",
   description="Bar chart showing the number of words per page (in raw response) per percentile (10, 25, 50, 75, and 90). The median mobile site displays 13.33% less text content than its desktop counterpart.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=831714745&format=interactive",
   sheets_gid="775602646",
   sql_file="pages_wpt_bodies_by_device_and_percentile.sql"
-  )
-}}
+) }}
 
+{# TODO(authors, analysts): These charts convey the absolute counts of words, but the written interpretations discuss the relative differences between raw/rendered and desktop/mobile. Is it worth having charts depict these relative differences directly? A reader might be thinking "where did these numbers come from". #}
```

Comment:
@aleyda @Tiggerito

Reply:
@rviscomi do you suggest changing the graph to reflect the % better and/or rather the absolute numbers also in the text? cc @Tiggerito

Reply:
I like that  🙂

Reply:
Hi @rviscomi - thanks for following up :) I agree that the usage of the tables is better in this case, we can definitely replace them - just let me know the best way to proceed. 

Regarding the difference in the median calculation: @ipullrank can you please take a look at this? Since it's a difference in the numbers it would be better that you take a look directly in case you might want to also change something in the analysis outcome.

Reply:
I went with a combined table that only focused on the medians:


Values | Desktop | Mobile | Difference
-- | -- | -- | --
Raw | 360 | 312 | -13.33%
Rendered | 402 | 348 | -13.43%
Difference | 11.67% | 11.54% |  

I've also updated @ipullrank's commentary to reference the numbers from the table.